e2e/wdio/core/tests/message-box.e2e-spec.ts

```diff
@@ -0,0 +1,84 @@
+import { MessageBoxPo } from '../pages/message-box.po';
+import {
+    click,
+    doesItExist,
+    getElementArrayLength,
+    getElementClass,
+    getText,
+    isElementDisplayed
+} from '../../driver/wdio';
+import {
+    iconsArr, buttonClassArr
+} from '../fixtures/appData/message-box'
+
+describe('Message-box test suits', function () {
+    const messageBoxPage = new MessageBoxPo();
+    const {
+        resultTxt, mobileExample, positionExample, messageBoxExample, basedObjectExample, openTemplateExample,
+        sematicTypesExample, basedComponentExample, complexTemplateExample, button, okButton, cancelButton, messageBox,
+        messageIcon
+    } = messageBoxPage;
+
+    beforeAll(() => {
+        messageBoxPage.open();
+    }, 1);
+
+    it('Should check working of message-boxes', () => {
+        checkMessageBoxWorking(basedObjectExample);
+        checkMessageBoxWorking(mobileExample);
+        checkMessageBoxWorking(positionExample);
+        checkMessageBoxWorking(basedComponentExample);
+        checkMessageBoxWorking(openTemplateExample);
+        checkMessageBoxWorking(sematicTypesExample);
+        checkMessageBoxWorking(messageBoxExample);
+        checkMessageBoxWorking(complexTemplateExample);
```

Comment:
In this case, better to modify selectors to not be example specific and have just 1 checkMessageBoxWorking()